dbt_copilot_helper/commands/dns.py

```diff
@@ -24,8 +25,14 @@
 # (run a test before removing)
 
 # Base domain depth
-MAX_DOMAIN_DEPTH = 2
+AVAILABLE_DOMAINS = {
+    "great.gov.uk": "live",
+    "trade.gov.uk": "live",
+    "prod.uktrade.digital": "live",
```

Comment:
Has this been moved over to `live`? This was still in `dev` the last time I used this.

Reply:
Don't know. I'll ask in the slack channel.

Reply:
Actually, I looked in the dev and live accounts and it is in the dev account.

Reply:
Yup, shall we change this back to `dev` for the time being?